NEWS.md

```diff
@@ -43,9 +43,11 @@ that need carrying the last observation forward other than `analysis_var`
 gained the `tmp_obs_nr_var` argument. It helps flagging or selecting consecutive
 observations or the first or last observation in a by group. (#1724)
 
-- `ADLB` metadata data set called `atoxgr_criteria_ctcv5` updated to remove unit check for
-`HYPERURICEMIA` as grade criteria based on `ANRHI` only.  This metadata holds criteria for lab grading
-based on [Common Terminology Criteria for Adverse Events (CTCAE) v5.0](https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm) (#1650)
+- The functions `derive_vars_merged`, `derive_var_merged_cat`, 
+`derive_var_merged_character`, `derive_var_merged_exist_flag`, 
+`derive_var_merged_summary`, and `derive_vars_merged_lookup` were updated to 
+allow renaming in the argument `by_vars` (#1680).
+
```

Comment:
Can you use () in all the functions.  This way the become links to the function documentation.

![image](https://user-images.githubusercontent.com/10111024/222311178-e945c924-6198-4117-805f-58ab4a32f1f9.png)

Reply:
Was the section about ADLB grading update deliberately removed?

Reply:
I think it got moved down - something happened in a merge and grading update has been getting moved around in every PR :(

Reply:
Updated this. Not sure about the ADLB grading update. I'll look into it.